src/src/targets.h

```diff
@@ -12,7 +12,7 @@
 /// General Features ///
 #define FEATURE_OPENTX_SYNC //uncomment to use OpenTX packet sync feature (requires OpenTX 2.4 onwards) - this reduces latency.
 #define LED_MAX_BRIGHTNESS 50 //0..255 for max led brightness
-
+// #define USE_UART2 // uncomment to use UART2 for MSP on the TX module
```

Comment:
Should we stick this in user_defines.txt?

Reply:
Good idea!